KVM/qemu/feature_test.cfg

```diff
@@ -12,6 +12,10 @@
             feature_dir_names = "cmpccxadd"
         - umip:
             feature_dir_names = "umip"
+        - lam:
```

Comment:
lam/lass are same platform (SRF)'s feature with cmpccxadd, so let them together. i.e. move lam/lass above of umip.